src/plot_api/plot_api.js

```diff
@@ -404,6 +404,11 @@ function opaqueSetBackground(gd, bgColor) {
     setBackground(gd, bgColor);
 }
 
+function blendSetBackground(gd, bgColor) {
+    var blend = Color.combine(bgColor, 'white');
```

Comment:
doesn't this end up visually equivalent to `opaqueSetBackground`? And if so can we remove `opaqueSetBackground`?